docs/topics/simulation_setup.rst

```diff
@@ -0,0 +1,182 @@
+quantity. Typically, defining the state requires performing MD simulations
+in each stable state to characterize its bounds, and there is not a unique
+appropriate definition when studying a particular problem of interest.
+
+In TPS, only the states need to be defined. In TIS, one must also define the
+interfaces.
+
+Defining good states
+--------------------
+
+In TPS, the main traits defining the path ensemble are the state
+definitions. The essential point is that states should be stable: a
+trajectory started from any point in the state should be expected to take a
+long time before visiting any other state.
+
+The topic of state definitions has been included in several review articles.
+For more, see:
+
+* `Bolhuis and Dellago. "Trajectory-based rare event simulations." in
+  Reviews Computation Chemistry vol. 27 (2010)
+  <http://onlinelibrary.wiley.com/doi/10.1002/9780470890905.ch3/summary>`_
+
+In TIS, the role of the minus move in decorrelating trajectories should also
+be considered. Ideally, a single minus move trajectory should be just long
+enough to fully decorrelate (lose all memory) within the state. This puts
+restrictions on both the volume of the state as well as that of the
+innermost interface.
+
+
+Defining good interfaces
+------------------------
+
+Much of the literature on TIS uses the expression that "interfaces foliate
+phase space." In the OPS picture, where interfaces are volumes instead of
+surfaces, this means that inner interfaces (closer to the state) must be
+fully contained by outer interfaces. In other words, for every volume in the
+ordered set of the state, interface 0, interface 1, etc.; any point in phase
+space that is inside interface :math:`i` is also inside interface :math:`i+1`.
+Importantly, this includes the state: the innermost interface must entirely
+encapsulate the state. Note that there is no way for OPS to check this for
+arbitrary state definitions; it requires user attention.
+
+Interface placement in TIS is essential to its efficiency. If interfaces are
+too far apart, the statistics in the estimate for
+:math:`P_A(\lambda_{i+1}|\lambda_i)` (see :ref:`tis-analysis`) are poor. If
+interfaces are too close together, then to span the transition, we'll need
+more interfaces. Since each interface represents an ensemble that must be
+sampled to convergence, this can badly hurt the efficiency. Typically, we
+aim for about 20-30% overlap between successive interfaces.
+
+There are approaches to `iteratively optimize the location of interfaces
+<https://doi.org/10.1063/1.3601919>`_.  These have been implemented for OPS,
+and will be added to the core code in the future.
+
+-----
+
+Getting an initial trajectory
+=============================
+
+Even with configurational sampling, initial conditions are not always
+completely trivial. A PDB entry might be missing hydrogens, and probably
+needs to be solvated (and energy-minimized). However, obtaining initial
```

Comment:
Add: "Also for MC of e.g. a liquid it is often easier to start with a lattice, that then can be melted: